firefox.god

```diff
@@ -1,5 +1,5 @@
 God.watch do |w|
   w.name = "Firefox"
-  w.start = "/usr/bin/firefox-dev -headless --marionette -profile /home/firefox/profile/"
+  w.start = "/usr/bin/firefox-dev -headless -marionette -profile /home/firefox/profile/"
```

Comment:
why it is changed?

Reply:
Reading from project we forked https://github.com/deepsweet/firefox-headless-remote/blob/master/entrypoint.sh#L11 . But both `--marionette` and `-marionette` works, but since we use `-headless` and `-profile` , I also changed this. Minor change.